src/current/_includes/releases/v23.2/v23.2.21.md

```diff
@@ -0,0 +1,22 @@
+## v23.2.21
+
+Release Date: March 6, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-21-sql-language-changes">SQL language changes</h3>
+
+- Since v23.2, table statistics histograms have been collected for non-indexed `JSON` columns. Histograms are no longer collected for these columns if the `sql.stats.non_indexed_json_histograms.enabled` cluster setting is set to `false`. This reduces memory usage during table statistics collection, for both automatic and manual collection via `ANALYZE` and `CREATE STATISTICS`. [#140144][#140144]
```

Comment:
i think this data type is called `JSONB`, AFAICT we only have that but not a `JSON` type: https://www.cockroachlabs.com/docs/v25.1/jsonb.html